networkx/drawing/nx_pylab.py

```diff
@@ -1440,6 +1444,25 @@ def draw_planar(G, **kwargs):
     draw(G, planar_layout(G), **kwargs)
 
 
+def draw_forceatlas2(G, **kwargs):
+    """Draw a networkx graph with forceatlas2 layout.
+    This is a convenience function equivalent to::
```

Comment:
Please leave a blank line in the doc_string after the first "one-liner". Otherwise the sphinx documentation uses both lines as the short description for links in the documentation to this function.